util/genbuild_h/genbuild_h.sh

```diff
@@ -74,7 +74,7 @@ printf "#define COREBOOT_VERSION_TIMESTAMP $DATE\n"
 printf "#define COREBOOT_ORIGIN_GIT_REVISION \"$GITREV\"\n"
 
 printf "#define COREBOOT_EXTRA_VERSION \"%s\"\n" "$COREBOOT_EXTRA_VERSION"
-printf "#define COREBOOT_MAJOR_VERSION %d\n#define COREBOOT_MINOR_VERSION %d\n" `git describe --match [0-9].[0-9]* | sed 's/\([0-9]\)\.\([0-9][0-9]*\).*/\1 \2/'`
+printf "#define COREBOOT_MAJOR_VERSION %d\n#define COREBOOT_MINOR_VERSION %d\n" `git describe --tags --match dell_optiplex_9010_v[0-9].[0-9]* | sed 's/dell_optiplex_9010_v//g' | sed 's/\([0-9]\)\.\([0-9]\+\).*/\1 \2/'`
```

Comment:
common base used DASHARO_VERSION and companions

Reply:
@miczyg1 where?

Reply:
https://github.com/Dasharo/coreboot/pull/201/files#diff-681a6968a038003bd2aca3ddf33fc693b471ff84bb63555064ee5f900062d09aR70

Reply:
NVM, it was hidden in the unchanged lines... https://github.com/Dasharo/coreboot/pull/202/files#diff-681a6968a038003bd2aca3ddf33fc693b471ff84bb63555064ee5f900062d09aR70

So this line basically should be unchanged

Reply:
Fixed https://github.com/Dasharo/coreboot/pull/202/commits/373c960472deb16f2a35967a012f6f194be161fb